example2/README.md

```diff
@@ -116,9 +118,9 @@ Then deploy the dashboard by executing:
 Store the device registry endpoint in a variable:
 
     export IOT_TENANT=$(oc project -q ).iot
-    export REG_URL=https://$(oc get route device-registry -o jsonpath='{ .spec.host }')
-    export HTTP_URL=https://$(oc get route iot-http-adapter -o jsonpath='{ .spec.host }')
-    export MQTT_URL=ssl://$(oc get route iot-mqtt-adapter -o jsonpath='{ .spec.host }')
+    export REG_URL=https://$(oc -n enmasse get route device-registry -o jsonpath='{ .spec.host }')
```

Comment:
Yes, this could be a problem. As this won't work for students. They won't have access to the "enmasse" namespace.

We need to fix this in another way.